http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -625,44 +628,64 @@ class DataprocInterpreter[F[_]: Parallel](
   def updateDataprocImageGroupMembership(googleProject: GoogleProject, createCluster: Boolean)(implicit
     ev: Ask[F, AppContext]
   ): F[Unit] =
-    parseImageProject(config.dataprocConfig.customDataprocImage).traverse_ { imageProject =>
-      for {
-        count <- inTransaction(clusterQuery.countActiveByProject(CloudContext.Gcp(googleProject)))
-        // Note: Don't remove the account if there are existing active clusters in the same project,
-        // because it could potentially break other clusters. We only check this for the 'remove' case.
-        _ <-
-          if (count > 0 && !createCluster) {
-            F.unit
-          } else {
-            for {
-              projectNumberOpt <- googleResourceService.getProjectNumber(googleProject)
-
-              projectNumber <- F.fromEither(projectNumberOpt.toRight(GoogleProjectNotFoundException(googleProject)))
-              // Note that the Dataproc service account is used to retrieve the image, and not the user's
-              // pet service account. There is one Dataproc service account per Google project. For more details:
-              // https://cloud.google.com/dataproc/docs/concepts/iam/iam#service_accounts
-
-              // Note we add both service-[project-number]@dataproc-accounts.iam.gserviceaccount.com and
-              // [project-number]@cloudservices.gserviceaccount.com to the group because both seem to be
-              // used in different circumstances (the latter seems to be used for adding preemptibles, for example).
-              dataprocServiceAccountEmail = WorkbenchEmail(
-                s"service-${projectNumber}@dataproc-accounts.iam.gserviceaccount.com"
-              )
-              _ <- updateGroupMembership(config.groupsConfig.dataprocImageProjectGroupEmail,
-                                         dataprocServiceAccountEmail,
-                                         createCluster
-              )
-              apiServiceAccountEmail = WorkbenchEmail(
-                s"${projectNumber}@cloudservices.gserviceaccount.com"
-              )
-              _ <- updateGroupMembership(config.groupsConfig.dataprocImageProjectGroupEmail,
-                                         apiServiceAccountEmail,
-                                         createCluster
-              )
-            } yield ()
-          }
-      } yield ()
-    }
+    for {
```

Comment:
diff looks bigger than reality becuz I removed the `parseImageProject` call cuz I don't think it's necessary

Reply:
Could you elaborate on why you think it is not necessary anymore?

Reply:
all it's doing is parsing the project out of a custom dataproc link and do stuff if that's successful, which most likely originally somewhere we're using the parsed project, but that's not the case in the current code